conbench/api/compare.py

```diff
@@ -227,7 +239,7 @@ def _get_a_result(benchmark_result_id: str) -> BenchmarkResult:
         return benchmark_result
 
     @maybe_login_required
-    def get(self, compare_ids: str) -> f.Response:
+    def get(self, compare_ids) -> f.Response:
```

Comment:
nit: add a type annotation here

Reply:
I tried to not change anything on that front, not sure what happened :)